### DISCUSSION TITLE
How to fix

### DISCUSSION THREAD

**Discussion by frmdeveloper** [socketio/socket.io#4165] [Q&A] [answered]
### in here error
![Screenshot_20211112-061333](https://user-images.githubusercontent.com/80396521/141382391-3a621fd5-38c5-4e60-83e6-9a4ea952ec30.png)

### In the browser and node-fetch it works
![Screenshot_20211112-061423](https://user-images.githubusercontent.com/80396521/141382435-1bc3cfe7-edac-46ad-a0ce-25ee78a06ef6.png)

### ?
![Screenshot_20211112-061504](https://user-images.githubusercontent.com/80396521/141382589-7a6f76f1-6c07-4181-b171-682f5ad7efb0.png)

**Comment by darrachequesne** [comment, marked as answer]
Hi! Could you please explain your issue?

See also: https://socket.io/docs/v4/troubleshooting-connection-issues/
